### PR TITLE
Add strict version check to h2o_flow()

### PR DESCRIPTION
**Pull request by navdeep-G**
* This PR adds a `strict_version_check` parameter to `h2o_flow`.
* This is needed to invoke H2O flow from a rsparkling instance if H2O versions do not match via the `h2o.init()` call.

### CODE REVIEW

File: r/rsparkling/man/as_h2o_frame.Rd [hunk 1]
```diff
@@ -1,4 +1,4 @@
-% Generated by roxygen2: do not edit by hand
+% Generated by roxygen2 (4.1.1): do not edit by hand
```
Comment (mmalohlava) [inline r/rsparkling/man/as_h2o_frame.Rd]: Is this change necessary?

Comment (navdeep-G) [inline r/rsparkling/man/as_h2o_frame.Rd]: Docs need to be updated since we added an extra parameter to `h2o_flow()`. I have re-updated the docs with the latest version of roxygen2.

**Comment by navdeep-G** [comment]
@jangorecki We can squash with merge in Github

**Comment by navdeep-G** [comment]
Okay to merge?
